frontend/scripts/react-components/tool-selector/top-cards/top-cards.component.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Heading from 'react-components/shared/heading';
+import Text from 'react-components/shared/text';
+import PropTypes from 'prop-types';
+import capitalize from 'lodash/capitalize';
+
+import 'react-components/tool-selector/top-cards/top-cards.scss';
+
+const TopCards = ({ setCommodity, setCountry, countryName, commodityName, step }) => {
+  const renderName = name => (
+    <Text as="span" size="lg" weight="bold">
```

Comment:
test this stuff with transifex, we might need to add a `notranslate`

Reply:
It looks fine when I check in Spanish. Do you imagine some other risks that will make us change it?

Reply:
ok no so i was just being careful, because react sometime crashes when unmounting texts.